RTN-004.tex

```diff
@@ -53,7 +52,312 @@
+
+In particular, delegates who are given accounts at the start of DP0.1 but who do not have the work-hours to start using them for a few months (due to prior commitments, teaching loads, etc.) will be accommodated. 
+Active, responsive delegates who are given accounts for DP0.1 may keep them, and will not need to reapply for DP0.2.
+
+\subsection{Support for Delegates}\label{ssec:res_sup}
+
+It is recognized that on-boarding resources and organizational infrastructure provided by Rubin Observatory will be necessary to enable equitable access for all delegates including novice science platform users (Section \ref{ssec:sel_equity}), and for DP0 delegates to obtain the benefits (Section \ref{ssec:del_bene}), fulfill the responsibilities (Section \ref{ssec:del_resp}) -- in particular, delegate actions (Section \ref{ssec:res_act}). 
+
+The full set of resources will be presented elsewhere, but would include, e.g., virtual live sessions and recordings of RSP demonstrations, user guides and documentation, and jupyter notebook tutorials.
+The organizational infrastructure will include online open forums for question-and-answer, issue resolution, and general discussion (e.g., Community.lsst.org), as well as regularly scheduled working groups and "office hours" run by Rubin staff.
+Opportunities will be provided for DP0 delegates to communicate with the CET about needed resources and support (e.g., periodic surveys, drop-in discussion sessions).
+
+DP0 delegates are responsible for engaging with the resources in order to learn about the RSP and the DP0 data set (Section \ref{ssec:del_resp}).
+Creating or enhancing DP0 resources and support infrastructure -- e.g., writing tutorial notebooks, leading a collaborative working group, helping other delegates resolve issues -- are all potential delegate actions (Section \ref{ssec:res_act}).
+
+\subsection{Example Delegate Actions}\label{ssec:res_act}
+
+As described in Section \ref{ssec:del_resp}, the most important responsibility of DP0 delegates is to learn how to do scientific analyses in the RSP, and then take action that informs or enhances the DMS, the RSP, or a Science Collaboration. 
+
+The objective is for these delegate actions to extend the benefits of DP0 beyond the limited number of DP0 participants that Rubin Observatory is able to support, and to maximize the impact of DP0 with respect to the goals of testing and informing development of the RSP and preparing the scientific community for early science (Section \ref{ssec:intro_goals}). 
+These actions are voluntary and will not be tracked.
+Group work is encouraged.
+It is left to individual delegates to decide what kind of actions best match their interests and expertise. 
+
+Examples of potential delegate actions include but are not limited to:
+\begin{itemize}
+\item participate in feedback surveys managed by Rubin Observatory
+\item provide a public Jupyter notebook demo of some science analysis
```

Comment:
"provide a public  demo of some science analysis, e.g Jupyter notebook based."
I would not insist on a demo analysis being in a Jupyter notebook.

Reply:
The whole list is just examples, not insisting on anything, but I changed that item to say "provide a public demo of some science analysis (e.g., Jupyter Notebook)".